src/unix/print.js

```diff
@@ -8,6 +8,18 @@ const getCommand = (pdf, options) => {
   if (options.printer) {
     command += " -d " + options.printer;
   }
+  // check for other options
+  Object.keys(options).forEach(function(key) {
+    if (key !== "printer") {
+      const value = options[key];
+      if (value !== false) {
+        command += " -o " + key;
+        if (typeof value !== "boolean") {
+          command += "=" + value;
```

Comment:
what about ```command = `${command} = ${value}`;```

Reply:
sure, no problem. Just wanted to keep the convention I found in that file ...